doc/api/addons.md

```diff
@@ -213,6 +213,24 @@ Addon developers are recommended to use to keep compatibility between past and
 future releases of V8 and Node.js. See the `nan` [examples][] for an
 illustration of how it can be used.
 
+
+## N-API
+
+N-API is a newer API for building native Addons.  It is independent from
```

Comment:
I would drop `newer` here. Otherwise, it will probably claim to be new for years to come.